registries/now_agents.hocon

```diff
@@ -0,0 +1,118 @@
+
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+{
+    "llm_config": {
+        "model_name": "gpt-4o",
+    },
+    "tools": [
+        # These tool definitions do not have to be in any particular order
+        # How they are linked and call each other is defined within their
+        # own specs.  This could be a graph, potentially even with cycles.
+
+        # This first guy is the "Front Man".  He is identified as such because
+        # he is the only one with no parameters in his function definition,
+        # and therefore he needs to talk to the outside world to get things rolling.
+        {
+            "name": "NowAgents",
+
+            # Note that there are no parameters defined for this guy's "function" key.
+            # This is the primary way to identify this tool as a front-man,
+            # distinguishing it from the rest of the tools.
+
+            "function": {
+                "description": "Assist the user with queries related to ServiceNow."
+                "parameters": {
+                    "type": "object",
+                    "properties": {
+                        "inquiry": {
+                            "type": "string",
+                            "description": """
+    An inquiry from a user.
+    """
+                        },
+                    },
+                    "required": ["inquiry"]
+                }
+            },
+            "instructions": """
+You are the `ServiceNow Agent` agent.
+Before using a tool, explain your reasoning.
+Use your tools to respond to the inquiry. Always start by retrieving the list of AI agents and determining the best one to use for this use case. After invoking an agent, wait 5 seconds and then use the retrieve message tool to get a response from servicenow.
```

Comment:
I don't think `gpt-4o` can wait 🙃 . I saw the CodedTool has some waiting logic, that's probably why it works. It means you can probably simplify this prompt and reduce the number of tokens.